packages/realm-server/tests/realm-server-test.ts

```diff
@@ -1667,7 +1667,9 @@ module('Realm Server', function (hooks) {
   module('various other realm tests', function (hooks) {
     let testRealmServer2: Server;
 
-    hooks.beforeEach(async function () {
+    hooks.before(async function () {
```

Comment:
we are mutating the state of the test realm in teh tests below. if we use a `before` instead of a `beforeEach` we run into the possibility of the test realm state from one test that mutates the realm bleeding into another test.

Reply:
> is this ok because realm 2 is not being mutated?

That's right. I see an alternative solution (for that closed db adapter bug) where we I switch back to `beforeEach` and, in the virtual network, remove the handler from the previous realms when we recreate them.

Reply:
Fixed in https://github.com/cardstack/boxel/pull/1306/commits/48206862054e1f4ee7b7b04d6af11b5ec3e4819d